src/index.js

```diff
@@ -37,248 +34,105 @@ export default function supernova(env) {
       data,
     },
     component: () => {
-      const [objectData, setObjectData] = useState(null);
-      const [styling, setStyling] = useState(null);
-      const [expandedState, setExpandedState] = useState(null);
-      const [linked, setLinked] = useState(false);
-      const [selectionState, setSelectionState] = useState([]);
-      const [transform, setTransform] = useState(null);
-      const [initialZoom, setInitialZoom] = useState(null);
       const layout = useStaleLayout();
       const model = useModel();
       const element = useElement();
       const Theme = useTheme();
       const options = useOptions();
-      const [opts] = useState(options);
       const rect = useRect();
-      const selectionsAPI = useSelections();
       const constraints = useConstraints();
-      const [selectionsAndTransform] = useState({
-        api: selectionsAPI,
-        setState: setSelectionState,
-        linked: false,
+      const translator = useTranslator();
+      const [preRenderData, setPreRenderData] = useState(null);
```

Comment:
So what is preRenderData?

Reply:
What `preRenderTree` returns. `objectData` did not make any real sense to me, at least this refers to where it comes from... But yeah maybe a better name for preRender then rename this accordingly.